### ISSUE TITLE
Address HR concern about SME participation

### ISSUE THREAD

**Issue by sstrassberg** [usds/SME-QA-Website#272]
**Legend**

**U**: Usability (Major, Minor, Cosmetic)
**C**: Complexity (High, Medium, Low)
**P**: Priority (Now, Next, Later)

U: Major; C: Medium; P: Next

- [ ] Review existing documentation
 - [ ] Determine if it can be presented with visuals to support text
 - [ ] Create visual
 - [ ] Get user feedback

**Comment by wslack** [comment]
@sstrassberg what does this mean?

**Comment by sstrassberg** [comment]
@wslack I believe this was regarding giving HR a better sense of SME time burden for each phase. I think we need to decide if what we have already satisfies this: https://smeqa.usds.gov/hiring-phases/getting-started/estimating-time-commitment/

**Comment by wslack** [comment]
Hmmmm...we had that image well before July. Which pilot did it come from?

**Comment by sstrassberg** [comment]
@wslack I don't think it's from a pilot, but from user testing the website with HR Specialists.

**Comment by wslack** [comment]
Closing given updates!
